avro-fastserde/src/test/java/com/linkedin/avro/fastserde/FastSpecificDeserializerGeneratorTest.java

```diff
@@ -688,6 +689,22 @@ public void shouldReadWithTypesDeletedFromReaderSchema(Boolean whetherUseFastDes
     Assert.assertEquals(new Utf8("abc"), getField(record, "field"));
   }
 
+  @Test(groups = {"deserializationTest"}, dataProvider = "SlowFastDeserializer")
+  public void testDeserializerWithDifferentNamespace(Boolean whetherUseFastDeserializer) throws IOException {
+    Schema writerSchema = Schema.parse(this.getClass().getResourceAsStream(
+        "/schema/removedTypesTestWithADifferentNamespace.avsc"));
+    GenericRecord object = new GenericData.Record(writerSchema);
+    object.put("field", "abc");
+
+    RemovedTypesTestRecord record = null;
+    if (whetherUseFastDeserializer) {
```

Comment:
can you add assert(RemovedTypesTestRecord.class.namespace() != "com.linkedin.avro.unknown") to make this extra clear ?

Reply:
Done.